doc/TSCC.rst

```diff
@@ -172,7 +191,7 @@ To look at the stdout of a currently running job: :code:`qpeek <jobID>`. To look
 :code:`qpeek` will no longer work.
 
 To delete a running or queued job: :code:`qdel <jobID>`. To delete all running or queued jobs:
-:code:`qstat -u <user> | cut -f1 | cut -f1 -d | xargs qdel`
+:code:`squeue -u <user> | cut -f1 | cut -f1 -d | xargs qdel`
```

Comment:
Should it be `xargs scancel` instead of `xargs qdel`?

Reply:
```suggestion
:code:`squeue -u <user> | cut -f1 | cut -f1 -d | xargs scancel`
```

Reply:
nevermind, according to @yli091230 in https://github.com/gymrek-lab/gymreklab.github.io/pull/59/files#r1546434065, we can use

```suggestion
:code:`scancel -u $USER`
```